contents/handbook/growth/sales/expansion-and-retention.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Expansion & Retention
+sidebar: Handbook
+showTitle: true
+---
+
+> The most fundamental thing you should do first is to get multiple contacts at the same place in case our champion leaves. This is the best way to derisk a customer leaving in the future.
+
+## Expansion
+
+AEs also do expansion at PostHog. This is because we are constantly on a sales footing with customers - for the most part, we don't do steady state account management with an arbitrary 10% uplift at renewal team. There are two primary ways we look to drive expansion with a customer:
+
+**Cross sell new products**
+- First you need to find out who cares about the new product - is it the existing team or the new team?
+- Then you need to find out what are they using now for other tools - surface this during the check in calls that you already have scheduled as part of onboarding.
+- At this point
+  - If it's already a mature product we have shipped, you should aim to show how the product complements what they _already_ are using in PostHog - don't just arbitrarily sell in a product for the sake of it. 
```

Comment:
Not sure if you want to add something here, but I'm very happy to look at creating some sales assets or comparisons for CS if needs be. There's also customer stories, etc, I can look at. 

Basically: If CS has asset needs at this point, I'm at their disposal and consider that work a priority.

Reply:
Thanks - I'd suggest maybe chatting to @simfish85 and @camerondeleone once you've finished up any last support bits you're working on to identify what would have the biggest impact here? (I'm not sure if it's here or in helping drive new revenue, for example)